RoadMap_VK/RoadMap_VK/Controllers/FriendsTableViewController.swift

```diff
@@ -48,60 +51,87 @@ final class FriendsTableViewController: UITableViewController {
     // MARK: - Private methods
 
     private func getOneUser(indexPath: IndexPath) -> FriendsItem? {
-        let firstChar = sectionsMap.keys.sorted()[indexPath.section]
-        guard let users = sectionsMap[firstChar] else { return nil }
+        let firstChar = frendsMap.keys.sorted()[indexPath.section]
+        guard let users = frendsMap[firstChar] else { return nil }
         let user = users[indexPath.row]
         return user
     }
 
     private func fetchFriends() {
-        networkService.getFriends { [weak self] result in
+        vkAPIService.fetchFriends { [weak self] result in
+            guard let self = self else { return }
```

Comment:
```suggestion
        vkAPIService.fetchFriends {  result in
        
```
тебе здесь weak self не нужен, и guard тоже, так как селфами тут уже не пользуешься и напрямую к реалм сервису обращаешься